.github/workflows/kb_sdk-actions.yml

```diff
@@ -0,0 +1,136 @@
+name: Basic kb-sdk tests
+
+on:
+  [push, pull_request]
+
+jobs:
+
+  test_kb-sdk_builds:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
```

Comment:
is it possible to test on mac/win? We definitely support mac, and I think we say we support win...?

Reply:
kb-sdk docs explicitly say that we don't support windows -- there are hardcoded paths in various files that mess things up when I tried running the tests on win. On the mac runner, I can't run the `kb-sdk test` step because there's an issue with setting up docker (docker is not available natively on the macOS runner, but is on the ubuntu test runners).

Reply:
I would like to add the macOS tests -- if/when docker is properly supported via GitHub Actions, it should be very simple just to add it to the actions file.